cmake-integration.el

```diff
@@ -273,48 +295,37 @@ Throws an error if the target is not an executable.
 If TARGET-NAME is not provided use the last target (saved in a
 'cmake-integration-current-target-name')."
```

Comment:
The docstring needs to be updated, since there is no `cmake-integration-current-target-name` anymore.

Reply:
fixed